### PR TITLE
Converted all reactstrap and bootstrap related code to react-bootstrap

### LINKED ISSUE #800: Convert all reactstrap and bootstrap related code to react-bootstrap
Requirement(s):
- The title of the issue explains it well!

### PR DESCRIPTION
**Pull request by AshuKV**
Fixes #800

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "leaflet": "^1.7.1",
     "mousetrap": "^1.6.5",
     "react": "^17.0.1",
-    "react-bootstrap": "^1.4.3",
+    "react-bootstrap": "^1.5.2",
```
Comment (anushbhatia) [inline package.json:19]: remove the boostrap

**Review by anushbhatia — CHANGES_REQUESTED** [review]
```
src/components/Dropdown/Dropdown.js
3:44:54 PM:   Line 4:3:  'DropdownToggle' is defined but never used  no-unused-vars
3:44:54 PM:   Line 5:3:  'DropdownMenu' is defined but never used    no-unused-vars
3:44:54 PM:   Line 6:3:  'DropdownItem' is defined but never used    no-unused-vars
3:44:54 PM: src/pages/Editor/sections/Settings/DownloadFileModal.js
3:44:54 PM:   Line 2:25:  'ModalBody' is defined but never used    no-unused-vars
3:44:54 PM:   Line 2:36:  'ModalFooter' is defined but never used  no-unused-vars

```

Deploy is faling here too please check.

**Review by AshuKV — COMMENTED** [review]
I have reviewed the changes. The PR can be merged now.

**Comment by anushbhatia** [comment]
Resolve the conflicts

**Pull request closed without merging**